docs/GettingStarted.md

```diff
@@ -141,6 +141,10 @@ either ``sinatra`` or ``sinatra/base``:
     require 'ddtrace'
     require 'ddtrace/contrib/sinatra/tracer'
 
+    Datadog.configure do |c|
+      c.use :sinatra, default_service: 'my-app'
```

Comment:
I find it strange that default service is not something like `'my-sinatra-app'` but that's not the subject of the PR. Just noticed.

Reply:
I just kept the previous value for consistency. All docs will be rewritten after when the new API will be available for all integrations. Thanks for pointing that!